_topic_maps/_topic_map.yml

```diff
@@ -266,6 +266,8 @@ Topics:
     File: installing-restricted-networks-gcp
   - Name: Installing a three-node cluster on GCP
     File: installing-gcp-three-node
+  - Name: Installation configuration parameters for GCP
```

Comment:
Since the assembly starts with "Before you deploy an OpenShift Container Platform cluster on Google Cloud Platform (GCP)", just wondering if this assembly should come before all of the assemblies that actually install it?

Reply:
> assembly should come before all of the assemblies that actually install it?

Not necessarily – but based on the current language, I did have the same thought when I read it. The goal of this PR (and the entire epic) is to simply refactor the existing content, allowing it to stand on its own. I am tracking several other small changes that I will aggregate into a single subtask for this work. One of which will be to give the abstract of this module a small copy edit. While it would be easy enough to fix here, I would prefer to track all of those changes in a single place.

Where this is purely reference material, I believe the current location is OK.